Dockerfile-build

```diff
@@ -78,13 +72,23 @@ RUN \
     git checkout tags/v2.4.7-1 && \
     scl enable devtoolset-7 'make CUDA8_GENCODE="-gencode=arch=compute_35,code=sm_35 -gencode=arch=compute_50,code=sm_50 -gencode=arch=compute_60,code=sm_60 -gencode=arch=compute_61,code=sm_61" -j src.build';
 
+#
+# Library versions
+#
+ARG python_version=3.7
```

Comment:
is idea to switch to 3.7 entirely?  Or is this just testing things out?  DAI still supports 3.6 only.